index.html

```diff
@@ -1,20 +1,37 @@
 <!DOCTYPE html>
-<html lang="pl">
+<html lang="en">
 <head>
-    <meta charset="UTF-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
-    <title>CodersCamp2020 | Star Wars API QUIZ</title>
-    <link rel="stylesheet" href="styles/App.css"/>
-
-    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-          rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GameSelection</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <script src="https://kit.fontawesome.com/84576b974f.js" crossorigin="anonymous"></script>
 </head>
-<body>
-
-<div id="swquiz-app">
-</div>
-
-<script src="src/index.js"></script>
+<body class="page">
+    <div class="menu">
+        <img class="menu__logo" src="img/StarWarsLogo.png" alt="">
+        <nav class="menu__list">
+            <a class="menu__item menu__item--selected" href="#">People</a>
+            <a class="menu__item menu__item--unselected" href="#">Vehicles</a>
```

Comment:
rozróżnienie wybranego elementu to dobry pomysł i praktyka, ale nie ma potrzeby dodawać klasy na unselected, bo to jest domyślne ustawienie - tworzymy niepotrzebnie tą klasę.

Reply:
ok, zatem zostaje tylko klasa selected